zerver/webhooks/gogs/doc.html

```diff
@@ -0,0 +1,44 @@
+<p>
+    First, create the stream you'd like to use for Gogs notifications, and then
```

Comment:
I was thinking about using the user documentation link instead of the direct link itself:
```html
First, <a target="_blank" href="../help/create-a-stream">create the stream</a> you'd like to use for Gogs notifications
```
You can remove the link in lines 5-6.